tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -1357,12 +1359,6 @@ public function customGroupCreate($params = array()) {
 
     $params = array_merge($defaults, $params);
 
-    //have a crack @ deleting it first in the hope this will prevent derailing our tests
-    $this->callAPISuccess('custom_group', 'get', array(
-      'title' => $params['title'],
-      array('api.custom_group.delete' => 1),
-    ));
```

Comment:
@eileenmcnaughton this was breaking the v4 test and on closer look appears to be broken. AFAIK this is incorrect chaining syntax (wrapping the chain param in `array()` shouldn't work) and therefore was not doing anything.

Reply:
@colemanw there are some weird & wonderful syntaxes that wok but it's easy to test - just do a PR removing just these lines against master

Reply:
Ok see https://github.com/civicrm/civicrm-core/pull/14239